### PR TITLE
Reduces cooldown of the Bluespace Artillery on the space battle away mission

### PR DESCRIPTION
**Pull request by Buggy123**
180 Byond seconds seems a little too long, considering this is one of the hardest away missions, this reduces it to a third, 60 Byond seconds.

I've never seen a group succeed at this mission before, and the only solo antag that could reasonably do it is a Changeling, and it would take 10-20 minutes at least. Since it is so incredibly difficult, getting essentially 2-3 shots in a 10 minute period seems a little unfair.

### CODE REVIEW

File: code/modules/awaymissions/bluespaceartillery.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 
 /obj/machinery/artillerycontrol
-	var/reload = 180
```
Comment (Iamgoofball) [inline code/modules/awaymissions/bluespaceartillery.dm]: defines fam tbh imo ttyl omg lol ftw

Comment (Buggy123) [inline code/modules/awaymissions/bluespaceartillery.dm]: This may have been the first time i've ever githubbed.

Or coded at all...

**Comment by Aranclanos** [comment]
what's your byond account?

**Comment by Buggy123** [comment]
Buggy1234

**Comment by Aranclanos** [comment]
ok just curious

File: code/modules/awaymissions/bluespaceartillery.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/awaymissions/bluespaceartillery.dm]: #DEFINE arty_reload_time 180
    ....
    var/reload = arty_reload_time
    ....
    src.reload > arty_reload_time

etc etc

Comment (Iamgoofball) [inline code/modules/awaymissions/bluespaceartillery.dm]: oh make sure arty_reload_time is uppercase

**Comment by Buggy123** [comment]
Updated slightly, reload time is now defined in one place as goofball suggested to make future editing easier.

**Comment by Iamgoofball** [comment]
:+1:

**Comment by Shadowlight213** [comment]
You madman.
:+1: x 73

**Comment by KorPhaeron** [comment]
I think doing 120 first would be more reasonable.

If you go overboard it will just get the mission disabled

**Comment by Buggy123** [comment]
Honestly I think 60 is the most reasonable. If you're going for pure destruction, you could release the singularity much more easily, which can eat a third of the station with remotely decent RNG.

One major difference is that the artillery is targetable by default, whereas the singularity requires a 14TC item to target and can't be targeted at all as anything other than traitor. However you have to be off-station to shoot it, and you can't hear comms from the station, so actually aiming for a person specifically is very difficult even with the cooperation of fellow antags.

**Comment by Jordie0608** [comment]
Can you change the define name to all-caps for distinction

**Comment by Robustin** [comment]
Do it.

60 is enough to actually make me care that someone is using it.

120 would only make sense if you could aim it using station cameras (DO IT).

**Comment by KorPhaeron** [comment]
>120 would only make sense if you could aim it using station cameras (DO IT).

Fuck I really want to do this now

**Comment by YotaXP** [comment]
It already takes more than 3 minutes for engineering to fix the damage from one shot.  Are you TRYING to get the shuttle called???

...you are, aren't you?

**Comment by Buggy123** [comment]
Hey, if anyone actually tried to fix stuff when it's reasonably fixable instead of calling the shuttle...

**Comment by Buggy123** [comment]
Minor update, changed the define name to all caps for consistency sake, as requested by Jordie.

**Comment by tkdrg** [comment]
Don't really see a problem with this, if 60 ends up being OP we can always just change a single number and bring it back up to 120 or whatever.

**Pull request merged**